tests/test_plugin.py

```diff
@@ -18,6 +19,9 @@ class NavigationSelectorTestCase(TestCase):
     def setUp(self):
         self.selector = NavigationSelector(None)
         self.request = RequestFactory().get("/")
+        self.user = factories.UserFactory()
```

Comment:
self.user isn't used anywhere, it could just be:

`self.request.user = factories.UserFactory()`